.github/workflows/smoke-test.yml

```
@@ -177,37 +178,37 @@ jobs:
           path: python/src/opendp/
 
       - name: Install package
-        run: python -m pip install -e ${{ matrix.extra-install }}
+        run: python -m pip install -e .[${{ matrix.extra-install }}]
 
       - name: Test
-        if: ! (contains(matrix.extra-install, 'polars') || contains(matrix.extra-install, 'mbi'))
         # There's not a "skipif" for doctests, so use "-k" to exclude based on name patterns.
         # (Even in unittests we should use "importskip" rather than "skipif".)
-        run: pytest -v -k "not lazyframe and not polars and not pca and not rst and not contingency and not synthetic"
-
-      - name: Test Polars
-        if: contains(matrix.extra-install, 'polars')
-        run: pytest -v -k "not contingency and not synthetic"
+        run: |
+          case "${{ matrix.extra-install }}" in
+            PLACEHOLDER)          pytest -v -k "not contingency and not synthetic and not lazyframe and not polars and not pca and not rst";;
```

I think you can do `""` here and use an empty string above too.
```suggestion
            "")          pytest -v -k "not contingency and not synthetic and not lazyframe and not polars and not pca and not rst";;
```